CIP-0006/CIP-0006.md

```diff
@@ -0,0 +1,157 @@
+---
+CIP: 6
+Title: Extended Metadata
+Authors: Markus Gufler <gufmar@gmail.com>, Mike Fullman <mike@fullman.net>
+Comments-URI:
+Status: Draft
+Type: Standards
+Created: 2020-07-20 original draft
+Updated: 2020-11-24 2nd key-pair for validation
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP defines the concept of extended metadata for pools that is referenced from the pool registration data stored on chain.
+
+## Motivation
+
+As the ecosystem around Cardano stake pools proliferate so will the desire to slice, organize and search pool information dynamically.  Currently the metadata referenced on chain provides 512 bytes that can be allocated across the four information categories ([delegation-design-specification Section 4.2)](https://hydra.iohk.io/build/790053/download/1/delegation_design_spec.pdf):
+
+| key           | Value                                |  Rules  |
+| ---           | ---                                  |  ---  |
+|  `ticker` | Pool ticker.  uppercase | 5 Characters Maximum, Uppercase letters and numbers |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool  | 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+
+Many additional attributes can be envisioned for future wallets, pool explorers, and information aggregators.  The proposal below outlines an initial strategy for capturing this extended metadata.
+
+## Specification
+
+### On Chain referenced (main) metadata file
+We define two more fields for the on chain referenced metadata file that references another json file on a url with the extended metadata.  The proposed metadata is as follows:
+
+| key           | Value                                | Rules  |
+| ---           | ---                                  | ---  |
+|  `ticker`       | Pool ticker.  uppercase              | 5 Characters Maximum, Uppercase letters and numbers  |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool| 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+| `extDataUrl` | A URL for extended metadata | optional, 128 Characters Maximum, must be a valid URL |
+| `extHashUrl` | A URL with the extended metadata hash | optional, 128 Characters Maximum, must be a valid URL |
```

Comment:
I *think* you don't mean `extHashUrl` but rather `extSigUrl`. This file contains the _signature_ of the data file, which can be verified with the `extVkey`.